### PR TITLE
Settings Info:UI created

### PR DESCRIPTION
**Pull request by DavidSurkov**
image
used react-hook-form data is validating. email will be coming from props. IMHO the ui part is complete

### CODE REVIEW

File: src/components/Sample/Sample.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import React, { FC } from 'react';
 import { useAppSelector, useAppDispatch } from 'redux/hooks';
 import { increment, decrement } from 'redux/reducers/sampleSlice';
 import type { RootState } from 'redux/store';
-import { TitleStyled } from 'Sample.styles';
 import { useTranslation } from 'react-i18next';
-import GoogleAuth from 'GoogleAuth/GoogleAuth';
+import GoogleAuth from '../GoogleAuth/GoogleAuth';
+import { TitleStyled } from './Sample.styles';
```
Comment (lyudad) [inline src/components/Sample/Sample.tsx:7]: why do you need any changes in Sample file? its just example

File: src/pages/settings-page/components/ContactInfo/ContactInfo.tsx [hunk 2]
```diff
@@ -1,12 +1,136 @@
...
+  border-radius: 5px;
+`;
+
+const StyledLabel = styled.label`
+  font-size: 24px;
+`;
+
+const SaveButton = styled.button`
+  display: block;
+  font-size: 24px;
```
Comment (lyudad) [inline src/pages/settings-page/components/ContactInfo/ContactInfo.tsx:33]: constants

File: src/pages/settings-page/components/ContactInfo/ContactInfo.tsx [hunk 3]
```diff
@@ -1,12 +1,136 @@
...
+  border-radius: 10px;
+  background-color: ${GREEN_COLOR};
+  color: ${WHITE_COLOR};
+  border-color: ${WHITE_COLOR};
+  margin: 0 auto;
+`;
+
+const Form = styled.form`
+  width: border-box;
+`;
```
Comment (lyudad) [inline src/pages/settings-page/components/ContactInfo/ContactInfo.tsx:45]: move styles to separate file

File: src/components/Sample/Sample.tsx (hunk 1 above)
Comment (DavidSurkov) [reply src/components/Sample/Sample.tsx:7]: The imports weren't working

File: src/pages/settings-page/components/ContactInfo/ContactInfo.tsx (hunk 2 above)
Comment (DavidSurkov) [reply src/pages/settings-page/components/ContactInfo/ContactInfo.tsx]: done

File: src/pages/settings-page/components/ContactInfo/ContactInfo.tsx (hunk 3 above)
Comment (DavidSurkov) [reply src/pages/settings-page/components/ContactInfo/ContactInfo.tsx]: done

**Pull request merged**